modules/tensor_mechanics/src/materials/ComputeAdmissibleState.C

```diff
@@ -0,0 +1,281 @@
+                                     "inelastic_models.  Default = '1 1 ... 1'.  This "
+                                     "parameter is set to 1 if the number of models = 1");
+  return params;
+}
+
+ComputeAdmissibleState::ComputeAdmissibleState(const InputParameters & parameters)
+  : ComputeFiniteStrainElasticStress(parameters),
+    _max_its(parameters.get<unsigned int>("max_iterations")),
+    _relative_tolerance(parameters.get<Real>("relative_tolerance")),
+    _absolute_tolerance(parameters.get<Real>("absolute_tolerance")),
+    _output_iteration_info(getParam<bool>("output_iteration_info")),
+    _perform_finite_strain_rotations(getParam<bool>("perform_finite_strain_rotations")),
+    _elasticity_tensor(getMaterialPropertyByName<RankFourTensor>(_base_name + "elasticity_tensor")),
+    _elastic_strain_old(getMaterialPropertyOld<RankTwoTensor>(_base_name + "elastic_strain")),
+    _strain_increment(getMaterialProperty<RankTwoTensor>(_base_name + "strain_increment")),
+    _inelastic_strain(declareProperty<RankTwoTensor>(_base_name + "combined_inelastic_strain")),
+    _inelastic_strain_old(
+        getMaterialPropertyOld<RankTwoTensor>(_base_name + "combined_inelastic_strain")),
+    _tangent_operator_type(getParam<MooseEnum>("tangent_operator").getEnum<TangentOperatorEnum>()),
+    _num_models(getParam<std::vector<MaterialName>>("inelastic_models").size()),
+    _inelastic_weights(isParamValid("combined_inelastic_strain_weights")
+                           ? getParam<std::vector<Real>>("combined_inelastic_strain_weights")
+                           : std::vector<Real>(_num_models, true)),
+    _cto(_num_models)
+{
+  if (_inelastic_weights.size() != _num_models)
+    mooseError("ComputeAdmissibleState: combined_inelastic_strain_weights must contain the same "
+               "number of entries as inelastic_models ",
+               _inelastic_weights.size(),
+               " ",
+               _num_models);
+}
+
+void
+ComputeAdmissibleState::initQpStatefulProperties()
+{
+  ComputeStressBase::initQpStatefulProperties();
+  _inelastic_strain[_qp].zero();
+}
+
+void
+ComputeAdmissibleState::initialSetup()
+{
+  std::vector<MaterialName> models = getParam<std::vector<MaterialName>>("inelastic_models");
+  for (unsigned int i = 0; i < _num_models; ++i)
+  {
+    StateUpdateBase * rrr = dynamic_cast<StateUpdateBase *>(&getMaterialByName(models[i]));
+    if (rrr)
+      _models.push_back(rrr);
+    else
+      mooseError("Model " + models[i] + " is not compatible with ComputeAdmissibleState");
+  }
+}
+
+void
+ComputeAdmissibleState::computeQpStress()
+{
+  // Nothing to update during the first time step
+  // ComputeQpStress is not called during the zeroth time step, so no need to guard against _t_step
+  // == 0
+
+  RankTwoTensor elastic_strain_increment;
+  RankTwoTensor combined_inelastic_strain_increment;
+
+  if (_num_models == 1)
+    updateQpStateSingleModel(elastic_strain_increment, combined_inelastic_strain_increment);
```

Comment:
OK, could it be this (my mistake).  In the not-single case, updateState's second slot (_inelastic_strain_increment[i_rmm]) is zero, but in the single case, it might not be zero(?)   I thought the default constructor would zero it, but maybe not?   All the other classes just set this parameter to something, but perhaps your Bison class uses the parameter before setting it?